cmd/livepeer/livepeer.go

```diff
@@ -552,7 +552,15 @@ func main() {
 				}
 				sm = rc
 			} else {
-				sm = pm.NewSenderMonitor(recipientAddr, n.Eth, senderWatcher, timeWatcher, n.Database, cleanupInterval, smTTL)
+				smCfg := &pm.LocalSenderMonitorConfig{
```

Comment:
Would it hurt avoiding duplicating this on line L628 and just initialising the config regardless but only use it  in those two cases?

Reply:
Fixed in bae3a83